new-frontiers-14/frontier-station-14
# GeneratorSystem: no eject on empty

## About the PR



Stops generating fuel when a generator is empty.

## Why / Balance


Bug.  https://discord.com/channels/1123826877245694004/1296452037093101578

## How to test


1. Spawn a solid fuel generator (PACMAN, SUPERPACMAN, HYPERPACMAN).
2. Try to eject, it should do nothing.
3. Insert fuel (plasma, uranium, bananium).
4. Try to eject, you should get fuel-grade material back.
5. Try to eject again, nothing should happen.

## Media


## Requirements



## Breaking changes


**Changelog**


:cl:
- fix: Solid fuel generators now no longer spawn fuel when ejecting while empty.
